src/v/kafka/server/handlers/handler_probe.cc

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "kafka/server/handlers/handler_probe.h"
+
+#include "config/configuration.h"
+#include "kafka/server/handlers/handler_interface.h"
+#include "prometheus/prometheus_sanitize.h"
+
+#include <seastar/core/metrics.hh>
+
+#include <memory>
+
+namespace kafka {
+
+handler_probe_manager::handler_probe_manager()
```

Comment:
Can we stick this into an existing `.cc` file. Not sure what makes the most sense. Maybe just `kakfa/server.cc` or `handler_interface.cc`?